velox/type/DecimalUtil.h

```diff
@@ -304,6 +304,22 @@ class DecimalUtil {
     }
   }
 
+  inline static std::optional<int128_t> computeValidSum(
```

Comment:
Is this function used only by aggregate functions? Seems could be put in functions/lib/aggregates/DecimalAggregate.h.

Reply:
Yes, I moved it to `DecimalAggregate.h`.